packages/examples/upgrade/static/ts/full/module.ts

```diff
@@ -41,7 +41,7 @@ export class TextFormatter {
 })
 export class Ng2HeroesComponent {
   @Input() heroes!: Hero[];
-  @Output() addHero = new EventEmitter();
+  @Output() addHero = new EventEmitter<void>();
   @Output() removeHero = new EventEmitter();
```

Comment:
While not necessary for the typings to pass compilation, I think this should be changed to:

```suggestion
  @Output() removeHero = new EventEmitter<Hero>();
```

to set a good example, and to provide better typing in the template.